custom_components/truenas/manifest.json

```diff
@@ -1,18 +1,14 @@
 {
-  "codeowners": [
-    "@sdwilsh"
-  ],
+  "codeowners": ["@sdwilsh"],
   "config_flow": true,
   "dependencies": [],
   "documentation": "https://github.com/sdwilsh/hass-truenas",
   "domain": "truenas",
   "homekit": {},
   "iot_class": "local_polling",
   "name": "TrueNAS",
-  "requirements": [
-    "aiotruenas-client==0.7.0"
-  ],
+  "requirements": ["aiotruenas-client==0.8.0"],
```

Comment:
Do you need 0.8.0 for this?  If yes, I can finish up #52 here instead of doing it here.